### PR TITLE
Created labeled container

### PR DESCRIPTION
**Pull request by jorgenherje**
Labeled container to obtain a html.Div with label and padding on content

- Based on wcc.Selector - html.Div instead of html.Details with open/close
- For margin, label and padding of content in container

---

image

### CODE REVIEW

File: react/src/lib/components/Layout/LabeledContainer/labeledcontainer.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.webviz-labeled-container {
+    color: #555;
+    margin-top: 0.5vh;
+    margin-bottom: 0.5vh;
+}
+
+.webviz-labeled-container:hover {
```
Comment (rubenthoms) [inline react/src/lib/components/Layout/LabeledContainer/labeledcontainer.css:7]: `.WebvizLabeledContainer`

File: react/src/lib/components/Layout/LabeledContainer/labeledcontainer.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.webviz-labeled-container {
+    color: #555;
+    margin-top: 0.5vh;
+    margin-bottom: 0.5vh;
```
Comment (rubenthoms) [inline react/src/lib/components/Layout/LabeledContainer/labeledcontainer.css:4]: `view height` sizes are no good practice here. Use `8px` or `16px` (multiple of 8)?

File: react/src/lib/components/Layout/index.ts [hunk 3]
```diff
@@ -9,6 +9,7 @@ import "./Dropdown/dropdown.css";
 import "./Frame/frame.css";
 import "./Header/header.css";
 import "./Label/label.css";
+import "./LabeledContainer/labeledcontainer.css"
```
Comment (rubenthoms) [inline react/src/lib/components/Layout/index.ts:12]: `labeled-container` / `labeled_container`?

File: webviz_core_components/wrapped_components/labeled_container.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    - className (string; optional):
+        Additional css class to apply
+
+    """
+
+    def __init__(
+        self, children: Any, label: str = "", className: str = "", **kwargs: Any
+    ) -> None:
+        super().__init__()
+        self.className = className + "webviz-labeled-container"
```
Comment (rubenthoms) [inline webviz_core_components/wrapped_components/labeled_container.py:26]: Class name see above. Moreover, if a class name is given, wouldn't the resulting `self.className` become `givenClassNamewebviz-labeled-container`?

**Review by rubenthoms — CHANGES_REQUESTED** [review]
LGTM :+1: Some small issues.

File: react/src/lib/components/Layout/index.ts (hunk 3 above)
Comment (jorgenherje) [reply react/src/lib/components/Layout/index.ts:12]: Chose `labeled-container`

File: react/src/lib/components/Layout/LabeledContainer/labeledcontainer.css (hunk 2 above)
Comment (jorgenherje) [reply react/src/lib/components/Layout/LabeledContainer/labeledcontainer.css:4]: `16px`

File: webviz_core_components/wrapped_components/labeled_container.py (hunk 4 above)
Comment (jorgenherje) [reply webviz_core_components/wrapped_components/labeled_container.py:26]: True - fixed

File: react/src/lib/components/Layout/LabeledContainer/labeled-container.css [hunk 5]
```diff
@@ -0,0 +1,9 @@
+.WebvizLabeledContainer {
+    color: #555;
+    margin-top: 16px;
+    margin-bottom: 16px;
+}
+
+.WebvizLabeledContainer:hover {
+    box-shadow: 0px 0px 1px 0.5px lightgrey;
+}
```
Comment (rubenthoms) [inline react/src/lib/components/Layout/LabeledContainer/labeled-container.css:9]: Add newline?

**Review by rubenthoms — CHANGES_REQUESTED** [review]

**Review by rubenthoms — APPROVED** [review]
LGTM

**Pull request merged**